x/data/msgs_test.go

```diff
@@ -85,13 +92,13 @@ func TestMsgSignDataRequest_GetSigners(t *testing.T) {
 	msg = &MsgSignData{Signers: nil}
 	require.Empty(t, msg.GetSigners())
 
-	msg = &MsgSignData{Signers: []string{"abcd"}}
-	require.Panics(t, func() {
-		msg.GetSigners()
-	})
+	msg = &MsgSignData{Signers: []string{""}}
+	require.Equal(t, "", msg.GetSigners()[0].String())
```

Comment:
I'm not sure I understand this change. Should we be checking for panic? What's the purpose of this check if we already check setting signers in lines 89-90?

Reply:
panics should never happen in GetSigners, as ValidateBasic is always called first. TBH, i think this test should just be removed, i dont think we have this type of test anywhere else.